### PR TITLE
GEOMESA-2515 Add -p to log dir creation

### PR DESCRIPTION
**Pull request by jahhulbert-ccri**
Signed-off-by: Andrew Hulbert <ahulbert@ccri.com>

### CODE REVIEW

**Comment by elahrvivaz** [comment]
@jahhulbert-ccri do you want this back-ported to 2.1/2.2?

**Comment by jahhulbert-ccri** [comment]
@elahrvivaz i will probably want to backport this to 2.1.x as well. I think that @amcdonaldccri had some other things that I might add before merging.

**Comment by jahhulbert-ccri** [comment]
oh yeah to 2.2 as well. So the use case here is supporting multiple users with access to the tools which are installed as an RPM. The first user to write to the log wins and the other users can't delete it. So we are going to make it based on the $USER var in linux by setting GEOMESA_LOG_DIR in the env. Perhaps we should do that by default in the tools somehow as well.

**Comment by elahrvivaz** [comment]
I think putting the log into the user's home dir by default might be confusing... I know in general I'm often struggling to find where (non-geomesa) log files end up. Seems like we have a pretty good ability to specify it per-user though, so I like the current approach.

File: geomesa-tools/bin/common-functions.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@ function setGeoLog() {
     export GEOMESA_OPTS="-Dgeomesa.log.dir=${GEOMESA_LOG_DIR} ${GEOMESA_OPTS}"
```
Comment (amcdonaldccri) [inline geomesa-tools/bin/common-functions.sh]: The export GEOMESA_OPTS needs to be set

Comment (elahrvivaz) [reply geomesa-tools/bin/common-functions.sh]: is there another change that needs to be made? this pr was already merged

**Pull request merged**